javascript/javascript2/week1/Spirit-animals.js

```diff
@@ -0,0 +1,53 @@
+
+//Spirit animals generator
+const animals = [
+    "The fullmoon wolf",
+    "The honest friendly dog",
+    "The secretive cat",
+    "The magical and mysterious crow",
+    "The spiritual guiding hawk",
+    "The courageous eagle",
+    "The peace-loving dove",
+    "The butterfly-effecting butterfly"    
+];
+const para = document.getElementById("animal-name");
+const button = document.getElementById("button");
+const input = document.getElementById("user-name");
+button.addEventListener("click", genAnimalNames);
+function genAnimalNames () {
+const randomNum = Math.floor(Math.random() * animals.length);
+const animalName = animals[randomNum];
+    !input.value? 
```

Comment:
the javascript option to do `if` `else` with `?` `:`, is for quick inline if clauses. Please use `if` and `else` for multiline if clauses